py-polars/polars/config.py

```diff
@@ -20,14 +20,38 @@ def _get_float_precision() -> int:
     return -1
 
 
+def _get_digit_group_separator() -> str:  # pragma: no cover
+    return ","
+
+
+def _get_digit_group_size() -> int:
+    return 3
+
+
+def _get_decimal_separator() -> str:
+    return "."
+
+
+def _get_trim_decimal_zeros() -> bool:
+    return False
+
+
 # note: module not available when building docs
 with contextlib.suppress(ImportError):
-    from polars.polars import get_float_fmt as _get_float_fmt  # type: ignore[no-redef]
-    from polars.polars import (  # type: ignore[no-redef]
+    from polars.polars import (  # type: ignore[no-redef]  # noqa
+        get_float_fmt as _get_float_fmt,
+        set_float_fmt as _set_float_fmt,
         get_float_precision as _get_float_precision,
+        set_float_precision as _set_float_precision,
+        set_digit_group_separator as _set_digit_group_separator,
+        get_digit_group_separator as _get_digit_group_separator,
+        set_digit_group_size as _set_digit_group_size,
+        get_digit_group_size as _get_digit_group_size,
+        set_decimal_separator as _set_decimal_separator,
+        get_decimal_separator as _get_decimal_separator,
+        get_trim_decimal_zeros as _get_trim_decimal_zeros,
+        set_trim_decimal_zeros as _set_trim_decimal_zeros,
```

Comment:
Better to use `import polars.polars as plr` and then use `plr.set_float_fmt` etc. I have it like that in a few places, apparently I hadn't updated this file.

Reply:
Done; good tidy-up 👍